arangod/GeneralServer/HttpCommTask.cpp

```diff
@@ -733,7 +733,7 @@ void HttpCommTask<T>::sendResponse(std::unique_ptr<GeneralResponse> baseRes,
       if (ec || !(s = self.lock())) {  // was canceled / deallocated
         return;
       }
-      LOG_TOPIC("5c1e0", DEBUG, Logger::REQUESTS)
+      LOG_TOPIC("5c1e0", INFO, Logger::REQUESTS)
```

Comment:
I don't think we want to see this in log level `INFO`. This is spamming logs a lot, so we should make it `DEBUG` again.

Reply:
the REQUESTS default log-level is FATAL